crates/net/network/src/manager.rs

```diff
@@ -366,7 +374,12 @@ where
     /// configured.
     fn delegate_eth_request(&self, event: IncomingEthRequest) {
         if let Some(ref reqs) = self.to_eth_request_handler {
-            let _ = reqs.try_send(event);
+            let _ = reqs.try_send(event).map_err(|e| {
+                if let TrySendError::Full(request) = e {
+                    debug!(target:"net", ?request, "EthRequestHandler channel is full!");
```

Comment:
this will dump the entire message to the log, if which could be massive.
can omitt this here.